web/docs/introduction/migrate-from-11.md

```diff
@@ -438,6 +438,322 @@ The migration functions provided below are written with the typical use cases in
 
 ##### Username & Password
 
+:::caution Users will need to migrate their password
+There is a breaking change between the old and the new auth in the way the password is hashed. This means that users will need to migrate their password after the migration, as the old password will no longer work.
+
+Since users can't reset their password by receiving an email when using the Username & Password auth method, you will need to provide a custom page to migrate the password.
```

Comment:
```suggestion
Since users can't reset their password by receiving an email when using the Username & Password auth method. One way to handle this is to inform them about the need to migrate their password (on the login page) and provide a custom page to migrate the password. 
```